src/Metadata/MetadataBase.php

```diff
@@ -133,6 +136,22 @@ protected static function getConstraints(): array
         ];
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected static function validate(array $data, Collection $constraints): void
+    {
+        static::traitValidate($data, $constraints);
+
+        // The TUF spec requires that all key IDs be unique.
+        // @todo Use Symfony's Unique constraint for this when at least Symfony
+        //   6.1 is required.
```

Comment:
Can we make an issue for this to avoid adding another `todo` without a follow-up issue

Reply:
Opened #317 for that.